pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -192,12 +195,11 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (result re
 	annotations := host.GetAnnotations()
 	if annotations != nil {
 		if _, ok := annotations[metal3v1alpha1.PausedAnnotation]; ok {
-			return reconcile.Result{Requeue: true, RequeueAfter: pauseRetryDelay}, nil
+			reqLogger.Info("host is paused, no work to do")
+			return reconcile.Result{Requeue: false}, nil
 		}
 	}
 
-	reqLogger := log.WithValues("Request.Namespace",
-		request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling BareMetalHost")
```

Comment:
Is it worth moving this back so that it is logged before any error from line 191?

Reply:
done